test/integration/helpers.js

```diff
@@ -0,0 +1,32 @@
+const WAIT_BETWEEN_SCREENSHOTS_MS = 100;
+
+export async function waitUntilScreenStable() {
```

Comment:
This function ensures the screen's pixels are no longer changing, before we run `toMatchImageSnapshot()`